syft/format/option.go

```diff
@@ -33,6 +35,8 @@ func ParseOption(userStr string) Option {
 		return TableOption
 	case string(CycloneDxOption), "cyclone", "cyclone-dx":
```

Comment:
now that we have a JSON flavor of CycloneDX we should probably rename `CycloneDxOption` to `CycloneDxXMLOption` and provide additional parser variants like `cyclone-dx-xml` and `cyclone-xml`